test/Allocs.jl

```diff
@@ -0,0 +1,34 @@
+module PProfAllocsTest
+
+import PProf
+import Profile
+using ProtoBuf
+
+using Test
+
+const out = tempname()
+
+@testset "basic profiling" begin
+    Profile.Allocs.clear()
+    Profile.Allocs.@profile sample_rate=1.0 begin
+        # Profile compilation
+        @eval foo(x,y) = x * y + y / x
+        @eval foo(2, 3)
+    end
+
+    # Write the profile
+    outf = PProf.Allocs.pprof(out=out, web=false)
+
+    # Read the exported profile
+    prof = open(io->readproto(io, PProf.perftools.profiles.Profile()), outf, "r")
```

Comment:
😮 what is this `->` syntax??

Reply:
Lambda/Closure/Anonymous Function

Reply:
oh, of course 🤦‍♂️ just hadn't seen it without parens I guess